echo-notifications/src/test/groovy/com/netflix/spinnaker/echo/notification/ManualJudgmentTemplateTest.groovy

```diff
@@ -159,16 +160,17 @@ class ManualJudgmentTemplateTest extends Specification {
     rendered == expected
 
     where:
-
+    // @formatter:off
     // Note, there's a whitespace after each new line of the `instructions` because of `Jsoup.parse(...)`.
-    // This is because the field is HTML parsed to plaintext. Which also can contain Slack's psudo-markdown flavor.
-    expected = '''\
-        Stage <SPINNAKER_URL/#/applications/testapp/executions/details/exec-id?refId=stage-id&step=1|stage-name> for *testapp*'s *exe-name* pipeline build #*12345* is awaiting manual judgment.
+    // This is because the field is HTML parsed to plaintext. Which also can contain Slack's sudo-markdown flavor.
```

Comment:
I think this is supposed to be
```suggestion
    // This is because the field is HTML parsed to plaintext. Which also can contain Slack's pseudo-markdown flavor.
```